### PR TITLE
Create safehaven-4-opensource.md

### PR DESCRIPTION
**Pull request by jinzishuai**

### CODE REVIEW

File: Disaster Recovery/safehaven-4-opensource.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{{{
+  "title": "SafeHaven 4: Open Source Components",
+  "date": "28-09-2016",
+  "author": "Shi Jin",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+# List of Open Source Components in SafeHaven-4
```
Comment (srveit) [inline Disaster Recovery/safehaven-4-opensource.md]: Change `#` to `###`

**Comment by srveit** [comment]
This needs to have more content.

**Comment by jinzishuai** [comment]
@srveit I am glad you responded so quickly. I was not sure if this procedure is going to work at all.

I am trying to get an URL for this document first before putting contents to do.

But now that I know you are actively working on these pull requests, I will work on it right away.

But the way, do you happen to know the would be URL of this document once it is merged?
I need to include that in the EULA document that we are releasing soon.

Thanks,
Shi

**Comment by jinzishuai** [comment]
@srveit  I have put in the proper contents to the document I am changing. Could you please review and merge? Thank you.

Shi

**Comment by srveit** [comment]
The URL will be https://www.ctl.io/knowledge-base/disaster-recovery/safehaven-4-opensource/

**Pull request merged**